projects/sw-appcache-behavior/test/automated-suite.js

```diff
@@ -50,43 +63,28 @@ describe('sw-appcache-behavior Test Suite', function() {
     return testServer.killServer();
   });
 
-  automatedBrowserTesting.getDiscoverableBrowsers().forEach(function(browser) {
-    let driver = null;
-
-    // Helper method to generate the text of an App Cache manifest with a
-    // specific version.
-    const generateManifestText = version => {
-      return `CACHE MANIFEST
-# Version: ${version}
-
-CACHE:
-../common.css
-
-NETWORK:
-*`;
-    };
-
-    // Before each sub-suite, initialize a new driver so that we can start with
-    // a fresh environment.
-    before(function() {
-      driver = browser.getSeleniumDriver();
-      driver.manage().timeouts().setScriptTimeout(TIMEOUT);
-    });
+  seleniumAssistant.getAvailableBrowsers().forEach(function(browser) {
```

Comment:
This will run on all browsers the library can find (Opera, Firefox, Chrome). Hopefully I'll get to add more browsers, but when I do this may break tests. Worth considering.